translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/ProjectManager/activities/ActivityProjectManager.java

```diff
@@ -44,11 +46,9 @@
 import org.wycliffeassociates.translationrecorder.utilities.Task;
 import org.wycliffeassociates.translationrecorder.utilities.TaskFragment;
 
-import java.io.BufferedOutputStream;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.OutputStream;
+import java.io.*;
```

Comment:
don't import .*

Reply:
@sarabiaj yeah, I know... It's just my ide does that, and I didn't pay attention